timestampvm/service.go

```diff
@@ -61,7 +54,10 @@ type GetBlockArgs struct {
 
 // GetBlockReply is the reply from GetBlock
 type GetBlockReply struct {
-	APIBlock
+	Timestamp json.Uint64 `json:"timestamp"` // Timestamp of most recent block
+	Data      string      `json:"data"`      // Data in the most recent block. Base 58 repr. of 5 bytes.
+	ID        string      `json:"id"`        // String repr. of ID of the most recent block
```

Comment:
If we change `ID`'s type to `ids.ID` then we can just have the marshaller deal with the conversion.